src/rpdk/core/contract/resource_client.py

```diff
@@ -166,6 +166,7 @@ def _update_schema(self, schema):
         self.read_only_paths = self._properties_to_paths("readOnlyProperties")
         self.write_only_paths = self._properties_to_paths("writeOnlyProperties")
         self.create_only_paths = self._properties_to_paths("createOnlyProperties")
+        self.insertion_order = self.get_metadata("insertionOrder")
```

Comment:
can we name it more explicit, something like `properties_without_insertion_order`